cmd/rest/main.go

```diff
@@ -30,3 +41,19 @@ func Run() error {
 	}
 	return nil
 }
+
+func connectDB(opt *cfg.Options) (*sql.DB, error) {
+	str := fmt.Sprintf("host=%v port=%v user=%v password=%v dbname=%v sslmode=disable",
+		opt.DBConfig.Host, opt.DBConfig.Port, opt.DBConfig.Username, opt.DBConfig.password, opt.DBConfig.DBName)
+
+	db, err := sql.Open("postgres", str)
+	if err != nil {
+		return nil, fmt.Errorf("sql.Open(): %w", err)
```

Comment:
It's better to use more meaningful text in such cases.
"sql.Open(): ..." -> "failed to open database connection: ..."

Reply:
The same about ping.

Reply:
Thanks for remark. 

[fix: minor fix](https://github.com/go-devs-ua/octagon/pull/4/commits/3daece7eb84c8e06b3868c16a023ca5ac1a22263)